setup-playbook.yaml

```diff
@@ -117,6 +117,15 @@
 - hosts: localhost
   roles:
   - create_certificates
+  tasks:
+  - name: Generate vsphere.conf
+    when: (vsphere_datastore is defined and vsphere_datacenter is defined and vsphere_network is defined and vsphere_resourcepool is defined)
```

Comment:
Kunne vi forenklet denne til bare `vsphere is defined`? Som en samle-variable, så man slapp den lange linja.

Reply:
Helt enig, men fant ingen måte å lage samlevariabler i nais inventory.

Reply:
Kan bytte til yaml inventories